service/history/timerBuilder.go

```diff
@@ -280,17 +289,21 @@ func (tb *timerBuilder) loadActivityTimers(msBuilder *mutableStateBuilder) {
 					EventID:     v.StartedID,
 					TimeoutType: w.TimeoutType_START_TO_CLOSE,
 					TimeoutSec:  v.StartToCloseTimeout,
-					TaskCreated: v.TimerTaskStatus == TimerTaskStatusCreated}
+					TaskCreated: (v.TimerTaskStatus & TimerTaskStatusCreatedStartToClose) != 0}
```

Comment:
This timer task status is on activity. it is possible to have two timer tasks ScheduleToClose and a HeartBeat for an activity, that is why bitwise operation is introduced. i am not sure what having Equals() would add value here? this is more readable in one shot to figure out this is a bitwise flag.